src/kaltura-player.js

```diff
@@ -650,6 +691,20 @@ class KalturaPlayer extends FakeEventTarget {
     }
   }
 
+  _onChangeSourceEnded(): void {
+    KalturaPlayer._logger.debug('_onChangeSourceEnded');
+    this._initAutoPause();
+    this._initTabVisibility();
+    this._initScrollVisibility();
+  }
+
+  _onPlayerReset(): void {
```

Comment:
I think you should set it on reset to make it clear, if there's not another reason for player reset

Reply:
Didn't quite understand the remark

Reply:
why don't you reset from API instead player reset event from playkit?

Reply:
ok, so it is the same remark Oren said.
I answered him the following:
Playkit calls reset in configure and KP calls reset in its own reset and also calls configure in the constructor and the configure.
So if I want to handle all places I need to do it in several places.
That way i am sure this is done in all player reset calls and this is full proof for future change in KP which someone might call configure / reset in other places